DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_configuration_checks.py

```diff
@@ -0,0 +1,88 @@
+import json
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.utils.test_utils.tb_utils import tb_get_all_devices
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+
+@pytest.mark.asyncio
+async def test_vlan_can_set_max_vlans(testbed):
+    """
+    Test Name: Maximum vlans for the interface
+    Test Suite: suite_functional_vlan
+    Test Overview: Test maximum number of vlans that can be set on interface
+    Test Procedure:
+    1. Initiate test params
+    2. Create bridge entity and set state to "up" state.
+    3. Enslave interface to the created bridge entity.
+    4. Insert interface to all VLANs possible (4094)
+    5. Verify interface is in all possible (4094) VLANs
+    """
+
+    max_vlans = 4094
+    #  1.Initiate test params
+    dent_devices = await tb_get_all_devices(testbed)
+    device = dent_devices[0].host_name
+    test_port = dent_devices.links_dict[device][1][0]  # get first port from config
+
+    #  2.Create bridge entity and set state to "up" state
+    out = await IpLink.add(input_data=[{device: [{"device": "br0", "type": "bridge", "vlan_filtering": 1}]}])
+    assert out[0][device]["rc"] == 0, f"Failed to create bridge.\n {out}"
+
+    out = await IpLink.set(input_data=[{device: [{"device": "br0", "operstate": "up"}]}])
+    assert out[0][device]["rc"] == 0, f"Failed to set bridge status to UP.\n {out}"
+
+    # 3. Enslave interface to the created bridge entity
+    await IpLink.set(input_data=[{device: [{"device": test_port, "master": "br0"},
+                                           {"device": test_port, "operstate": "up"}]}])
+    assert out[0][device]["rc"] == 0, f"Failed to enslave links to bridge.\n {out}"
+
+    # 4. Insert interface to all VLANs possible
+    cmd = f"time for i in {{1..{max_vlans}}}; do bridge vlan add vid $i dev {test_port}; done"
+    await dent_devices[0].run_cmd(cmd)
+
+    # 5. Verify interface is in all possible(4094) VLANs
+    out = await BridgeVlan.show(input_data=[{device: [{"device": test_port, "cmd_options": "-j"}]}])
+    vlans = json.loads(out[0][device]["result"])
```

Comment:
Please add `assert out[0][device]["out"]`.
Please use `parse_output=True` instead of parsing json yourself.